src/packages/@ncigdc/modern_components/ExploreCasesTable/ExploreCasesTable.js

```diff
@@ -84,7 +84,7 @@ export default compose(
             arrangeColumnKey="exploreCases"
             total={cases.hits.total}
             endpoint="case_ssms"
-            downloadTooltip="Export All Except #Mutations and #Genes"
+            downloadTooltip="Export All Except #Mutations, #Genes and Slides"
```

Comment:
why do Mutations and Genes have a `#`, but Slides does not?

Reply:
What makes you sure it must `literally be "number of"`?

No being sceptical, genuinely curious

Reply:
Hehe it just occurred to me when I saw your comment. But not actually based in logic particularly...tbh I'm not sure why those columns would have it and not the others

Reply:
this is a hilarious convo :P Reason for the # will have to ask @phuongmy